[4777][ADD] auth_oauth_force_oauth (#60)
qrtl/thc-oca: auth_oauth_enforced/readme/DESCRIPTION.md

> This module controls login methods by email domain, requiring OAuth for
> specified domains, as defined in company settings.

Reviewer: ```suggestion
This module enforces OAuth login for users with specific domains listed in the company
settings.
```

> This module controls login methods by email domain, requiring OAuth for
> specified domains, as defined in company
> settings.

Reviewer: Let's avoid random line breaks.
```suggestion
This module controls login methods by email domain, requiring OAuth for specified
domains, as defined in company settings.
```
